package.json

```diff
@@ -5,6 +5,7 @@
     "packages/*"
   ],
   "dependencies": {
+    "node-fetch": "^2.6.7",
```

Comment:
Why are we adding this dependency? Please double-check all changes introduced in PRs, _especially_ anything touching package.json or yarn.lock. This is particularly important with larger PRs like this one, where individual file changes can be missed.